tests/dialects/test_presto.py

```diff
@@ -1071,6 +1071,19 @@ def test_presto(self):
                 "databricks": "REGEXP_EXTRACT('abc', '(a)(b)(c)', 0)",
             },
         )
+        self.validate_all(
+            "CURRENT_USER",
+            read={
+                "presto": "CURRENT_USER",
```

Comment:
One nit upon viewing this again, it's superfluous to add both (i.e `read` and `write`) Presto directions here; This is because `validate_all(query, read, write)` will make sure that:

- All `read` queries map to Presto's `query` _and_ 
- `query` will map to all `write` queries

That means we're essentially testing Presto (read) -> Presto (query) -> Presto (write), when just one direction would suffice.

Reply:
All feedback is welcome 😄
I've been looking at it for 30 minutes or so and I think I finally understand.
So because the test class has its dialect set to presto, I can at least remove the write test case for it then.
What I really want to test is that, after being parsed, it remains unchanged.

I took inspiration from some of the other test cases above it so there's probably some more slimming down to be done. I'll remove my redundant test case.